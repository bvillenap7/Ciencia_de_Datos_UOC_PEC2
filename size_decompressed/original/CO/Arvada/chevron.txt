We thought we were looking at a kite until it started to move strangly.;2007-06-09 12:45:00;4 minutes;We thought we were looking at a kite until it started to move strangely. Object was seen in the eastern sky east of Wadsworth, in Arvada, CO.  I thought it looked like someone was flying a kite.  It was hovering like a kite, but not in a place you would usually noticed a kite, like where there are trees and houses.  My husband said that no, that is not a kite.   The object pointed itself southward and then due west across Wadsworth Blvd and headed west towards the mountains.  We were in the car and were unable to track it from that point.  We got a decent look at it.  It was boomerang shaped, metallic and glistening. It seemed to be at a low altitude, small plane in size. ((NUFORC Note:  Witness elects to remain totally anonymous  provides no contact information.  PD));39.8289982352941;-105.110888235294
Metallic red then silver, chevron/boomerang shaped.  ((anonymous report));2016-10-15 15:30:00;15 minutes;Metallic red then silver, chevron/boomerang shaped. Looked at through binoculars.  Did not move in straight line. Shifted slowly in different directions.  Metallic silver and red changes in color.  Minimal clouds moving just above it.  Mostly clear blue skies.  Then suddenly disappeared. ((NUFORC Note:  Witness elects to remain totally anonymous provides no contact information.  PD));39.8289982352941;-105.110888235294
