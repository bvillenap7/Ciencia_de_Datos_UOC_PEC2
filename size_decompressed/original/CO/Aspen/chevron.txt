New Years Eve Anomoly In Aspen Colorado;2010-01-01 00:30:00;2 mins;New Years Eve Anomoly In Aspen, Colorado This was just after new years, us four were driving home from a party, and our car stopped working about 10 feet from the house.  Everything stopped when i turned they keys the car wouldn't make a noise.  It was very quiet and then we got out of the car and looked up. We saw two 15-20 feet long chevron shaped flying objects.  I was scared out of my mind. They were still were rimmed with blue and red and green and yellow lights.  It was a full moon and they had a distinct shape due to the bright sky of the full moon they were black but because of the lights, they kind of glowed they were moving slowly and down toward us.  Then something must have startled them but they moved north at a speed i have never seen anything go, and in a instant, the car turned back on, and the car alarm went off as well. My miles in my car are now set to zero, which is cool, but my radio doesn't work and neither does any of our cell phones.  I think there was some sort of magnetic force messing up our electronics.  Oh and my watch reset itself  too. ((NUFORC Note:  Witness indicates that date is approximate.  PD));39.1977833333333;-106.825375
