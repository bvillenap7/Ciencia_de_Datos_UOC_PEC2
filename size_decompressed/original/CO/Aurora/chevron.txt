Small Chevron shaped craft hovered silently above me at 200 ft. then zigged and zagged NW across Aurora.;1977-07-30 03:00:00;2 - 3 minutes;Small Chevron shaped craft hovered silently above me at 200 ft. then zigged and zagged NW across Aurora. Walking on Highline Canal path near Hinkley High School I noticed a small blinking light in the night sky.  At first I thought it was the light of a commercial aircraft at 20,000 ft. until it jiggled slightly up and down.  When the peripheal of my vision opened around the light I realized the light was at the bottom of a rectangular white craft slilently hovering approx. 200 ft. above me. Just above the tree line and above an apartment building. I stopped dead in my tracks, Rubbed my eyes and realized it was not an illusion.  The bottom of the craft as well as the sides were flat. The back side and front of the craft were angled. Much like the Chevron shape. It hovered motionlessly (except for slight jiggling up and down) over me for at least one minute.  Before it started off I could hear a slight air disturbance, and then it zigged slowly in a left angle about 50 - 75 ft. and stopped, then it zipped to a right angle and stopped, then left and stopped, then right and stopped in this exact pattern moving northwest acrossed Aurora towards the mountains until I could no longer see it. ((NUFORC Note:  Date may be approximate.  PD));39.6695444272445;-104.803991486068
Stingray like glide ghost.;2012-01-13 23:20:00;4 seconds;Stingray like glide ghost. On 1/13/2012 just about an hour ago from this post my mother and I saw a weird stingray but big, cloudy blueish object glide past her windshield of the vehicle.  My son and his friend were in backseat but dis not see anything.   I thought I was seeing things because I knew it was no bird, plastic bag or anything.  Temperture right now is 6 degrees no bird wld be flyimg in this. It came from the elementry schools direction NW and flew across with fluid ease & gone.   I said,  What was that?  My mother said, You saw it too, it looked like a stingray ghost!    We were in a residential area around the corner from the house so I IMMEDIATELY began to google and found this sight & hope to get answers.   Whatever it was seemed perfectfully harmless & seemed to fly across at the perfect moment to be seen.  I hope to find more people who have seen this.;39.6695444272445;-104.803991486068
Chevron-shaped object flyinf south over Aurora, Colorado.;2015-11-01 18:45:00;30 seconds;Chevron shaped object flyinf south over Aurora, Colorado. While standing in my backyard I was looking up in the sky and I saw a chevron shaped object with very dim light pattern on it ( seven total with thre on each side and one in the front ) flying at a low altitude from the north to the south.   It did not make any noise at all, but It was for real and could not believe what I saw. ((NUFORC Note:  We spoke via telephone with the witness, and he corrected the time to 1845 hrs. local time.  PD));39.6695444272445;-104.803991486068
Chevron shape objects at night.;2019-04-15 20:44:00;10 seconds;Chevron shape objects at night While out in our hot tub, my wife and I were stargazing, when we saw a Chevron-shaped craft with dim lights on the bottom of the wings 4 on each side and 2 at the front of Chevron.  It was moving from west to east and was completely silent and was visible for 5-7 seconds and then disappeared into the night.  I reported the same sighting on this website in early November of 2015 also.  The craft was for real!  We scanned the sky for the next 15 minutes and also saw another Chevron shaped object that was white heading south to north and this one did not have any lights on the bottom of the wings.;39.6695444272445;-104.803991486068
