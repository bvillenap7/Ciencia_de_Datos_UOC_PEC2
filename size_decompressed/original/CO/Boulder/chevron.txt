Low flying, silent, muted white lights on a chevron shaped glider;2005-10-03 21:30:00;45 seconds;Low flying, silent, muted white lights on a chevron shaped glider 2 adults and 1 8 year old girl were walking on a bikepath in Boulder, CO, coming home from dinner.  I spotted a swift, silent, glider-ike object flowing through the air at a very low altitude (less than 2000 feet).  It had 5 'lights' on it, in a zig-zag shape arrangment.  The lights were not spotlights, typical airplane lights, nor were they flashing.  They had a muted glowing quality, almost like a white/yellow glow in the dark look.  The 5 lights, I assumed at first were all on the same craft, but the far two lights wavered and moved closer together and then assumed the original shape again as it glided out of our sight.  It was very low and very large, a fist across when held at arms length.  We walked on, in amazement talking to each other and a voice on the path said, Did you guys see that?--our neighbor Alec had seen it too and he had planted his self down in the middle of the bike path.  He had noticed it when he said the 5 or 6 lights were seperate and only after they had moved apart and then converged did they take on the glider like/chevron shape when we had spotted it. The reason I am reporting this is not only my interest and excitment for UFO's but also that I saw the same thing less than a year ago, I was also in Boulder. I called the Boulder Municipal Airport the following morning askng if their were gliders up in Boulder County that previous nights, the man informed me that all gliders are grounded prior to sunset.  I am very interested to see if anyone else saw what we did and what the theories are?  I have a science background and consider myself an observant, relatively sane thinker.  This object was not a typical aircraft.  Most notably the craft was silent.;40.0217727272726;-105.258365102639
Pink and blue pulsating manta ray shaped ufo over boulder;2011-09-14 23:00:00;4 seconds;pink and blue pulsating manta ray shaped ufo over boulder I was at a park in central downtown Boulder, my boyfriend and I were just watching the stars on top of the hill in the park.  This strange 'manta ray' shaped craft appeared and just flew gracefully through the sky, from the North East to the South West.  I call it a manta ray shaped craft while it honestly looked like a manta ray was just soaring through the sky.  It was so peaceful, it glowed a faint pink and blue. Imagine the changing colors of the fish of the deep, it was exactly like those colors, and exactly like the pulsating motion.  The wings of the ship slightly appeared to be moving, and it left a faint light trail (Which we actually believed was just an illusion, while it traveled so fast). It was unbelievable, we couldn't believe how much it looked like a manta ray. It just soared so gracefully through the sky for a good 4 seconds.  It appeared to be far up in the atmosphere, based on its faint color, but it also seemed so huge.  We thought we were cra! zy at first, until we realized that we saw the same thing... we are just two college kids, no drugs were in our system... and we saw something in the sky that night, which still haunts us in a very mystical way today. ((NUFORC Note:  Witness indicates that the date of the sighting is approximate.  PD));40.0217727272726;-105.258365102639
Seven blue square lights seen moving in a chevron formation over Boulder, CO;2014-10-12 23:00:00;10 seconds;Seven blue square lights seen moving in a chevron formation over Boulder, CO I was along the railroad tracks near the power plant in eastern Boulder, CO.  Something caught my eye and I looked up to witness seven square pale blue lights moving in a Flying V pattern in a Southern direction.  At first I thought it may have been a gaggle of geese in formation, until I realized that the objects were evenly spaced, square in shape, and glowing a full pale blue.  I'm guessing the objects were 200 feet off the ground and moving 70 mph.  No sound could be heard. ((NUFORC Note:  Witness elects to remain totally anonymous provides no contact information.  PD));40.0217727272726;-105.258365102639
Dark Silent Chevron.;2018-05-15 23:16:00;10 seconds;Dark Silent Chevron. A very dark, perfectly silent chevron shape with what seemed like 2 very dim lights on each end of the wings was flying at a moderate speed seemingly only a couple hundred yards off the ground.  Only saw the dark shape move across the dark sky for about 8-10 seconds before it disappeared behind houses.  Flew extremely straight, and away from where I was standing.  No blinking on the dim lights.;40.0217727272726;-105.258365102639
