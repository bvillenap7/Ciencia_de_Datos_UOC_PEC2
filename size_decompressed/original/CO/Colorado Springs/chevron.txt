No lights much larger than any aircraft very high rate of speed over 10000 ft. AGL;2004-09-13 20:30:00;30sec.;No lights much larger than any aircraft very high rate of speed over 10000 ft. AGL I spoted a very large object flying east SE over my house for less than a minuute on 13 Sep. 2004 aftrer the object disapeared over clouds a small aircraft on near the same rout flew over at that time I realized how large the object must have been and how fast it must have been traveling and at what altitude.;38.8753097510373;-104.772105082987
I went down to Colo Sprgs for a work conference on Sun, 6/24/18. I was staying at a hotel near I-25 and Circle Dr. This hotel had patio;2018-06-24 21:30:00;1 minutes;I went down to Colo Sprgs for a work conference on Sun, 6/24/18.  I was staying at a hotel near I-25 and Circle Dr.  This hotel had patios and balconies for smokers or people who just wanted to sit outside.  The room I stayed in was on the first floor, near the tennis/basketball courts.   Anyway, I had just gotten back from having dinner with a friend of mine and I was starting to wind down for the night.  The drive down there from Denver was horrible because we encountered major storms earlier in the day.  Lots of heavy rains and hail from Denver to the Sprgs.  So I decided to go out on the patio and have a smoke or two before I went to bed. I was out there for about 5 or 10 mins, when I suddenly saw a white translucent chevron aircraft flying silently to the South/Southwest.  I couldn't believe my eyes!  I have never seen one of these objects before.  It was flying around 2000-3000 ft. up in the air and it was moving very fast.  Unfortunately, there was a large tree close to where I was standing and the object flew behind the tree and I didn't see it again. As soon as I noticed it was gone for good, I noticed several perfect circles (a bit larger than basketballs) that were also white and translucent that were much lower in the sky, and they were much closer to me and my position!  At first, I thought my eyes were playing tricks on me, so I started to rub them.  But as soon as I stopped rubbing them, I saw more of the white circles just hovering in the air. There were people playing tennis that night and there were other people on one of the other courts because I could hear them being really loud and laughing about something, but I didn't know what.  I didn't get the impression that any of them saw what I just saw, but I don't know for sure.  I did not sleep well that night at all and I kept getting the impression that something was watching me.  It was so creepy!  I have had numerous UFO sightings over the past 21 years, but I've never seen anything like these objects.;38.8753097510373;-104.772105082987
