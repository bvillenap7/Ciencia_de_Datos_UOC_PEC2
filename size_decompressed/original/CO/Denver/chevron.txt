I was driving home down 13th street, I saw a object hovering over 13th street very low, at first I could not tell how low as I was appr;1999-12-29 17:50:00;10 minutes;I was driving home down 13th street, I saw a object hovering over 13th street very low, at first I could not tell how low as I was approaching it. It had a blue light then a red light then a blue light. That was all I could see of the object as I was appraching it at about 35 mph. I began to wonder what it was because it was so low and seemed to be hovering unmoving. I eventually ruled out plane because of the complete stillness of lights. I ruled out crane or building lights for low flying aircraft because there was nothing above two stories in a 10 block radius. I decided as I got closer it must be a hovering helicopter.I was coming up to the lights now and I could tell they were only about 6 stories high so I was expecting to see a helicopter. As I passed under the object I saw a greyish black boomerang shaped object.My mouth hung open and I said aloud to myself what in the #### is that! I could not stop in my tracks as there were cars behind me. I turned aff at the nest street came to a park postion and jumped out of my car. It was not longer then 30 seconds since I saw the object but it was gone . I jumped in my car after surveying the skyline went up to twelfth went up three blocks and came down thirtenth again.There was nothing above thirtennth or low flying craft anywhere that I could see. I came around again for a second pass and again nothing above thirteenth and nothing anywhere in the sky that I cold see except plane high above (very very high) I then realized something also paculiar about the obect although it had red an blue lights (which made me originally think it was normal air traffic )I realized at the third pass when I saw the other plane high up that the object I saw lights had never blinked once any of them. I had watched it for about 2 minutes coming up to it and not on light ever blinked.;39.7388117431815;-104.985074484367
Large boomerang-shaped craft seen during meteor storm in the Denver, Colorado area.;2001-11-16 02:45:00;One Minute;Large boomerang-shaped craft seen during meteor storm in the Denver, Colorado area. During the Leonid meteor storms, a witness described in great detail a boomerang shaped craft larger than a commercial airliner that was viewed for a full 15-20 seconds as it crossed the horizon. It was viewed for another thirty seconds or so until it left the observer's field of vision. It was described as being dark, dull-rust in color with several lights on it's underside.;39.7388117431815;-104.985074484367
V formation of white lights move with no sound;2003-10-13 21:20:00;45 seconds;V formation of white lights move with no sound v formation of multiple white lights (maybe 10) moved across the sky with no sound. It was about the size of the top of our regular brick chimney. It moved a little faster than an airliner across the sky with no sound. First saw it as it traveled south to north, lost sight of it behind a tree then it returned north to south 5-10 seconds later. this was too fast for an airliner to do a U turn. traveling north it was a little smaller and in more of an arc shape, traveling south it was larger (lower?) and in more of a V shape. it may have had more than one row of lights (we dident agree on that) We noticed that a few minutes afterward there seemed to be an unusual number of jets in the sky, though they may have been airliners and we are 20 miles from Denver international. I am an aerospace engineer and had no prior knowledge of the previous sightings of similar light patterns.;39.7388117431815;-104.985074484367
My sister and I grew up in a model aviation building/flying family and we both have a keen knowledge of the state of aviation art in fl;2006-10-02 17:00:00;30 secs;My sister and I grew up in a model aviation building/flying family and we both have a keen knowledge of the state of aviation art in flying machines. This experience gave me the knowledge and encouragement to become a leader in the aerospace science and international airline spaceline operations. By the grace of God I currently lead the industry as the developer of a spaceship that competed successfully in the 2004 X-Prize race to space for civilian operators. My sisters possessed this same general background in aviation to confirm that what she saw flying object just below the clouds headed directly south as if head toward Colorado Springs Colorado. While looking up into the local airspace of an extremely located south east Denver neighborhood she observed  a new type of flying ship that was shaped like a open V apexes first configuration and moving very fast while making no sound emissions at about 5:30 pm. Because of its smaller size, as compared to a major airliner, she believes that the object, which flew directly over her head at about a 90 degrees zenith point could have possible been a large UVA, CUVA or single piloted craft. What captured  her attention was the fact that  the flying machine was following an  unconventional flight route that deviated from the normal east/west ATC regulated  traffic patterns class D airspace  routing support for both DIA and our Buckley AFB  flight patterns. She observed white lights illuminate near the outer section of both delta wing tip leading edges and possessed no power-plant thrust production smoke or vapors trailing it as it few at a low altitude. She does not remember the exact day but it was seen early last week around the 2ed of October. One final note in relation to this strange sighting its interesting to note that the sighting was made just before the North Korea nuclear boom test firing event schedule which place Americaâ€™s spying activities on full alert. My sister know her airplane and I believe what she saw was something new. ((NUFORC Note:  Witness elects to remain totally anonymous  provides no contact information.  PD));39.7388117431815;-104.985074484367
A boomerang-shaped black object flys overhead during an eclipse in Denver.;2014-04-15 01:15:00;10 seconds;A boomerang-shaped black object flys overhead during an eclipse in Denver My friend and I were watching an eclipse a little while ago when we saw this really big boomerang shaped thing start to fly over us.  You really had to look hard to see it.   At first I thought it was a huge bird but it got closer and you could tell it wasn't an animal.  It was black and blended into sky, and had white lights or dots on it but they weren't very bright.  It silently and slowly went right over the house.  It seemed like it was very close.   We paused for a moment because we thought maybe someone was flying a kite.  I went around the house to see where it went but I couldn't see it.  It didn't seem like anyone was flying a kite nearby.  It was a clear night.   My friend and I are 22, and we've known each other all our lives.  I am very skeptical, I don't know if it is likely that this was some sort of alien thing but I think its fun to entertain the idea.   My friend is more skeptical than I am and she and I were both a little weirded out by the experience.  I feel a little shaken up by it, and I wanted to tell the story because I just wanted it to be on record somewhere.   Denver is just not a place I ever expected to see a UFO, especially in a residential area.  I also know that I can't ever tell the story again because no one will believe it. ((NUFORC Note:  Witness elects to provide no contact information.  PD));39.7388117431815;-104.985074484367
V-shaped craft seen over Denver.  Also, saw fast gray wispy barrel looking craft moving thru low hanging fog.;2014-10-24 22:30:00;1 hour;V shaped craft seen over Denver.  Also saw fast gray wispy barrel looking craft moving thru low hanging fog 10/24/14 Friday 10:30 I was in the backyard when I saw a blackVwith 4 glowing yellow lights going down the 2 outer sides.  Going from my 1 o'clock position east to west.  I snapped out of it after seeing it for about 5 seconds & ran inside to get to get my husband.  We ran out to the front yard.  It had disappeared. We went to the backyard and after 5 minutes we both saw it again.  It appeared at the eve of the roof going north to south this time.  But it had undulating wings made of the same yellow colored lights.  6 of them so 3 on either side.  They were gently moving like a snake out from the center.  But the lights moved independently from either side kinda like a crawling octopus tentacle.  It lasted about 10 seconds that time. It reappeared again about 15 minutes later 12 o'clock high moving northeast to southwest.  My husband saw it first.  It was very high this time.  It was like the first one l saw again.  It lasted about 4 seconds that time. I saw another one about 10 minutes later.  It was completely different.  It looked like a wispy rod shaped gray cloud.  Either that or it occurred to me laster while writing this, that it could have been a gray long barrel shaped craft with no lights pushing thru the clouds.  It moved from 12'oclock  high to the direct south.  It was very fast again.  It lasted 3-4 seconds.  My husband didnt see that one. Then l saw the same type of wispy cloudlike thing but it was triangle shaped.  No lights.  Very strange floating motion.  It was just gliding like on ice.  It lasted about 3 seconds.  I was facing west.  It appeared   above me going west. The fifth one was 15 minutes later.  My husband had gone inside.  It was another wispy rodlike cloud going from 12:00 high to the west.  It was fast, lasting 3 seconds.;39.7388117431815;-104.985074484367
Boomerang craft-stealth bomber like- w/round amber lights;2015-10-15 20:00:00;2 minutes;Boomerang craft- stealth bomber like- w/round amber lights I live on the edge of north downtown Denver.  Last night, I saw a large stealth bomber looking craft with orange lights moving across the sky, quietly, at a fluid like speed.  It was traveling south west.   Minutes later, I saw another craft traveling south at higher altitude, quietly and even faster.  I kept wiping my eyes thinking I was losing my mind.  A few minutes later another craft flew by at a high rate of speed, on the same path as the first.   I completely feel like a coo-coo bird telling people about what I saw but it actually happened.  Not sure if this is some new government craft or what.  Out of this world technology.   Plan on taking video tonight to see if it happens again. ((NUFORC Note:  Witness elects to remain totally anonymous provides no contact information.  PD));39.7388117431815;-104.985074484367
Dark chevron/boomerang object.;2015-11-02 19:00:00;<1 minute;Dark chevron/boomerang object Saw a chevron/boomerang shaped craft gliding southbound toward city park.  It was silent and couldn't have been flying higher than 250 feet, difficult to tell since it was such a barely visible object with no lights. ((NUFORC Note:  Witness elects to remain totally anonymous provides no contact information.  PD));39.7388117431815;-104.985074484367
