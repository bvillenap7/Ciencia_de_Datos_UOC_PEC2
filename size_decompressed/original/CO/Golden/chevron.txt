Several hazy images, seemingly attached solidly in space, in a V/Chevron shape.;2002-10-09 22:00:00;15 seconds;Several hazy images, seemingly attached solidly in space, in a V/Chevron shape. I was taking a quick jog around the block, and investigating the sound of a helocopter and a police pulled over car.... But as the helocopter flew west, out of sight and sound to the mountains above Denver, I looked up at the sky.... A vague set of contrasted shapes, in the shape of a V, seemingly flowing the path of the helocopter's. Basically, they seemed to be the reflection of city lights shining on the bottom of some very distant bird formation. But, as I focused the shapes interacted uniformly, as if they were fixed on the front of a Stealth Bomber (flying wing)... making a slight turn and straightening manuvre but with no complete form... just the fixed, hazy dots. I thought maybe some geese were flying over silently, and as they passed I would see birds but it flew way too fast and silent. The tipping of the form, uniformly, alarmed me that birds don't usually make such manuvres. Also, just after the slight manuvre, they passed over an active/lit-up baseball field, and I stared, to see if I would see a V of birds fly through the lights. But they were much higher than they seemed. Because they went out of view in the light, this showed great velocity.;39.7598265306123;-105.220002040816
I saw this only the next day as I was looking at the photos I took with my IPhone SE 2020. It must be a glitch in the camera. Could a s;2020-12-09 19:00:00;Unknown;I saw this only the next day as I was looking at the photos I took with my IPhone SE 2020. It must be a glitch in the camera. Could a satellite show up this way?;39.7598265306123;-105.220002040816
